diff --git a/cmd/osmosisd/cmd/root.go b/cmd/osmosisd/cmd/root.go
index 4a81bb715..dba1830d1 100644
--- a/cmd/osmosisd/cmd/root.go
+++ b/cmd/osmosisd/cmd/root.go
@@ -129,7 +129,7 @@ var (
 		{
 			Section: "osmosis-mempool",
 			Key:     "max-gas-wanted-per-tx",
-			Value:   "60000000",
+			Value:   "80000000",
 		},
 		{
 			Section: "wasm",
@@ -683,7 +683,7 @@ func initAppConfig() (string, interface{}) {
 	}
 
 	DefaultOsmosisMempoolConfig := OsmosisMempoolConfig{
-		MaxGasWantedPerTx:         "60000000",
+		MaxGasWantedPerTx:         "80000000",
 		MinGasPriceForArbitrageTx: ".1",
 		MinGasPriceForHighGasTx:   ".0025",
 		Mempool1559Enabled:        "true",
